app/content/carto3-workspace/settings/configuring-your-organization.md

```diff
@@ -1,23 +1,11 @@
 ## Configuring your organization
 
-CARTO 3 Workspace includes several options for configuring your organization. From the *Settings* menu, you can rename your organization, manage who can join your organization, view the details of your organization plan, register existing apps, invite users to join your organization, and delete your organization.
-
-### Renaming your organization 
```

Comment:
Are we getting rid of this section by mistake or on purpose?

Reply:
on purpose ;)